<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FullCalibrationDialog</class>
 <widget class="QDialog" name="FullCalibrationDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>411</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri Light</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Sample skip</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="modeAnalog">
       <property name="text">
        <string>Analog</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="modeDigital">
       <property name="text">
        <string>Digital</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="modeCombined">
       <property name="text">
        <string>Combined</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QComboBox" name="inputRangeView">
     <item>
      <property name="text">
       <string>100 mV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>250 mV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500 mV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1000 mV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2000 mV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5000 mV</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10000 mV</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QSpinBox" name="calibrationDuration">
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QPushButton" name="load">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Calibration time</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Channels</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="ch1label">
     <property name="text">
      <string>CH1</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="4">
    <widget class="QPushButton" name="startCalibration">
     <property name="text">
      <string>Calibrate selected</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Calibration mode</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="channelSelection0">
       <property name="text">
        <string>CH1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="channelSelection1">
       <property name="text">
        <string>CH2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="channelSelection2">
       <property name="text">
        <string>CH3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="channelSelection3">
       <property name="text">
        <string>CH4</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="2">
    <widget class="QSpinBox" name="ao3">
     <property name="minimum">
      <number>-32768</number>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="currentStage">
     <property name="text">
      <string>CH1(0.25V)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QSpinBox" name="sampleSkip">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="ch4label">
     <property name="text">
      <string>CH4</string>
     </property>
    </widget>
   </item>
   <item row="15" column="2" colspan="2">
    <widget class="QPushButton" name="apply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QSpinBox" name="do2">
     <property name="minimum">
      <number>-32768</number>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QPushButton" name="save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QSpinBox" name="do0">
     <property name="minimum">
      <number>-32768</number>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Analog offset</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QSpinBox" name="ao1">
     <property name="minimum">
      <number>-32768</number>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QSpinBox" name="ao2">
     <property name="minimum">
      <number>-32768</number>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QSpinBox" name="do3">
     <property name="minimum">
      <number>-32768</number>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Digital offset</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="ch3label">
     <property name="text">
      <string>CH3</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="ch2label">
     <property name="text">
      <string>CH2</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QSpinBox" name="do1">
     <property name="minimum">
      <number>-32768</number>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QSpinBox" name="ao0">
     <property name="minimum">
      <number>-32768</number>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="rangeSingle">
       <property name="text">
        <string>Viewed</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rangeAll">
       <property name="text">
        <string>All</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="0" colspan="4">
    <widget class="QLabel" name="hintLabel">
     <property name="text">
      <string>Terminate the selected channels before starting calibration.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>View input range</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Calibrate ranges</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Channel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
